build/linux/arch/PKGBUILD

```diff
@@ -15,7 +15,7 @@ changelog=
 
 source=("https://github.com/LedgerHQ/ledger-live-desktop/archive/v${pkgver}.tar.gz"
         "ledger-live.desktop")
-md5sums=('5dc6bd1e8d6dedcfe760d1c75565aad8'
+md5sums=('832e4525de10cfc07f28213ddacf1cdf'
```

Comment:
The checksum has changed for 1.1.4

Reply:
ye (but the PKGBUILD is for 1.1.2 https://github.com/LedgerHQ/ledger-live-desktop/blob/6625ae77ea6041ecec6bf5a7395e3ca852de3573/build/linux/arch/PKGBUILD#L5)

This one can be closed now btw